<a href="https://colab.research.google.com/github/Du-nara/ME421-Mechanical-Systems-Lab-A3/blob/main/Heat%20Transfer_new/E_20_388_Heat_Transfer.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## Part 1: Propose a method to find out the feasibility of the available unit.  

Objective

---

Determine whether the Heat Engine Laboratory Air Conditioning (AC) unit can:
1. Cool 40 milk containers from 35–40 °C to 15 °C within 2 hours
2. Maintain milk at 15 °C for ~24 hours


\
Step 1: Calculation of the Required Cooling Load

---

**Known Data:**\
Number of containers = 40

Per container:

*   Milk mass = 5 kg
*   Container mass = 2 kg (Aluminum)

Specific heats:


*   Milk: $C_{p,\text{milk}}$ = 4000J/kg⋅K
*   Aluminum: $C_{p,\text{Al}}$ = 921.1J/kg⋅K

Target cooling:

*   $T_{initial}$ = 35–40°C
*   $T_{final}$ = 15°C

Worst case used for design:

*   $T_{initial}$ = 40°C
*   ΔT= 40−15 =25K

Heat Removal per Container:

Milk cooling load,

<center>
$Q_{milk}$ = m$C_{p}$ ​ΔT

$Q_{milk}$ = 5×4000×25

$Q_{milk}$ ​= 500,000J
</center>


Container cooling load,
<center>
$Q_{Al}$ = 2×921.1×25

$Q_{Al}$ = 46,055J
</center>

Total per container,
<center>
$Q_{container, total}$ = 546,055J
</center>

Total Heat Removal (40 containers),
<center>
$Q_{total}$ = 40×546,055

$Q_{total}$ = 21,842,200J

$Q_{total}$ ≈ 21.84MJ
</center>
Required Cooling Rate,

*   Cooling time = 2 hours = 7200 s

<center>
$\dot{Q}$ ​= Q/t

$\dot{Q} = $$\frac{21.84 \times 10^{6}}{7200}$

$\dot{Q}$ ​≈ 3033W

$\dot{Q}$ ≈ 3.03kW
</center>

Minimum refrigeration capacity required ≈ 3.1 kW.

(excluding safety margin)

Step 2: Apply Safety Factors,

Include:

*   Heat gains from walls
*   Door opening losses
*   Air infiltration
*   Fan & lighting loads
*   Milk loading disturbance

Typical safety margin:
<center>
1.2–1.5
</center>

Assume 1.4:
<center>

$Q_{design​}$ = 3.03×1.4

$Q_{design​}$ ≈ 4.24kW

</center>

Design cooling capacity required ≈ 4.3 kW

Step 3: Compare with AC Unit Capacity

Obtain AC nameplate data:

*   Cooling capacity (kW or BTU/hr)

Conversion:
*   1kW = 3412BTU/hr

Feasibility Criteria:

| Condition | Interpretation |
|----------|----------|
| AC < 3 kW       | Cannot meet 2-hour cooling        |
| AC ≈ 3–4 kW     | Marginal       |
| AC > 4.3 kW   | Suitable       |      

\
Step 4: Evaluate Temperature Maintenance Load

After reaching 15 °C:

Only heat gains must be offset:

Sources:

*   Transmission through walls
*   Air leakage
*   Door opening
*   Warm container addition (if any)

Method:

<center>
$Q_{maintenance​​}$ =UA( $T_{ambient​}$-$T_{room​​}$)
</center>

Where:
*   U = Overall heat transfer coefficient
*   A = Surface area
*   ΔT = Temperature difference

If:

$Q_{maintenance​​}$ < $AC_{capacity}$

Temperature can be maintained.


Step 5: Experimental Verification Plan

Run controlled test in lab:

Procedure:

1.   Place thermal mass equivalent to milk load
(water containers simulate milk)
2.   Measure:
*   Room air temperature vs time
*   Product temperature vs time
*   Compressor duty cycle
*   Power consumption

3. Validate:
*   Time to reach 15 °C
*   Steady-state temperature stability

Step 6: Final Feasibility Decision

The AC unit is feasible if:
*   Analytical cooling load ≤ AC capacity
*   Experimental cooling time ≤ 2 hours
*   Temperature maintained at 15 °C
*   Compressor not overloaded

Summary of Proposed Method

1. Compute cooling load
2. Apply safety factor
3. Compare with AC capacity
4. Estimate maintenance load
5. Perform experimental test
6. Make feasibility conclusion






























## Part 2: Experiment with the cooling unit in the Heat Engine Laboratory and obtain the necessary data.  







In [ ]:
!pip install PyPDF2

   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 232.6/232.6 kB 5.5 MB/s eta 0:00:00


In [ ]:
from google.colab import files
uploaded = files.upload()

Saving E_20_388_Observations.pdf to E_20_388_Observations.pdf


In [ ]:
import PyPDF2

with open("E_20_388_Observations.pdf", "rb") as file:
    reader = PyPDF2.PdfReader(file)
    for page in reader.pages:
        print(page.extract_text())

Observation Sheet  
Table  1: Experimental Observation Sheet for Cooling Unit Performance  Evaluation  (Gas Side 
Temperature Measurements ) 
Time 
(min) Air 
Flow 
Rate 
(kg/hr)  Room 
Return  
Air 
Temp 
T3 
(°C) Condenser 
Inlet 
Temp T1 
(°C) Condenser 
Outlet 
Temp T2 
(°C) Evaporator 
Inlet Temp 
T4 (°C)  Evaporator 
Outlet 
Temp T5 
(°C) Refrigerant 
Pressure 
Low Side 
(MPa ) Refrigerant 
Pressure 
High Side 
(MPa)  
0 30 31.3  44.6  32.7  10.2  11.2  0.3 1.8 
1 30 31.2  44.3  32.8  10 11.5  0.3 1.8 
2 30 31.4  44.7  32.4  10.1  10.5  0.3 1.8 
3 30 31 44.4  32.3  10.2  11 0.3 1.8 
4 30 31.2  44.5  32.2  9.7 11 0.3 1.8 
5 30 31 44.1  32 9.6 10.8  0.3 1.8 
6 30 31.3  44.3  31.9  9.5 11.5  0.3 1.8 
7 30 31.4  44.4  32.1  9.1 10.2  0.3 1.8 
8 30 31.3  44.4  31.8  9.6 10.6  0.3 1.8 
9 30 31.1  44.3  31.6  9.7 10.6  0.3 1.8 
10 30 31.1  44.1  31.5  9 10 0.3 1.8 
11 30 31.1  44.5  31.6  9.2 10.5  0.3 1.8 
12 30 30.9  44.8  31.7  8.8 10.6  0.3 1.8 
 
  Table  2: Air Side Psychrometric 

## Part 3: Determine the developments required to be introduced for the unit if required to store 80 milk containers per day.

Step 1 – Recalculate Cooling Load for 80 Containers

---

Previously for 40 containers:

Milk per container = 5 kg

Container mass = 2 kg

For 80 containers:

$m_{milk}$ ​= 80×5 = 400 kg

$m_{Al}$ ​= 80×2 = 160 kg

Temperature Reduction

Worst case cooling:

40°C → 15°C

Δ𝑇=25𝐾

Heat Removal from Milk

<center>$Q_{milk}$​= m$C_p$​ΔT

$Q_{milk}$​= 400×4000×25

$Q_{milk}$​= 40,000,000 J

$Q_{milk}$​= 40 MJ </center>

Heat Removal from Aluminum Containers

<center>$Q_{Al}$ ​= 160×921.1×25

$Q_{Al}$ ​= 3,684,400 J

$Q_{Al}$ ​= 3.68 MJ </center>

Total Cooling Load

<center>$Q_{Total}$ ​= 40+3.68

$Q_{Total}$ ≈ 43.68 MJ</center>


Step 2 – Required Cooling Capacity

---

Cooling must be completed within 2 hours:

t=7200 s

<center>$\dot{Q} = $$\frac{43.68 \times 10^{6}}{7200}$

$\dot{Q}$ ≈ 6067 W

$\dot{Q}$ ≈ 6.07 kW</center>

Add Safety Factor (1.3–1.5)

Assume 1.4:

6.07×1.4 = 8.5 kW

Required Cooling Capacity ≈ 8–9 kW

Step 3 –Compare with Existing Unit

---

From Part 2 observations:

Refrigerant: R22

Mass flow rate: 30 kg/hr

Suction pressure: 0.3 MPa

Discharge pressure: 1.8 MPa

Evaporator inlet/outlet refrigerant temperatures ≈ 9–11°C

$\dot{m}$ = 30 kg/hr

Convert to kg/s: $\dot{m}$ = 0.00833 kg/s

Low side pressure = 0.3 MPa

Evaporator refrigerant temperature ≈ 10°C

For R22:

At 0.3 MPa:

Saturation temperature ≈ 0–2°C

Latent heat $h_{fg}$ ≈ 210kJ/kg

Since measured temperature ≈ 10°C, refrigerant is slightly superheated.

from R22 tables at 0.3 MPa:

$h_{in}$ (after expansion, saturated mixture) ≈ 230 kJ/kg

$h_{out}$ (slightly superheated vapor) ≈ 410 kJ/kg

So:

<center>Δh = $h_{out}$ - $h_{in}$

Δh≈410−230

Δℎ≈180𝑘𝐽/𝑘𝑔

Δh≈180kJ/kg</center>

Cooling Capacity Calculation

<center>$\dot{Q}_{evap}$ = $\dot{m}$ Δh

$\dot{Q}_{evap}$ = 0.00833×180,000

$\dot{Q}_{evap}$ = 1499 W

$\dot{Q}_{evap}$  ≈ 1.5 kW</center>

Step 4: Interpretation

---

<center>$\dot{Q}_{actual}$  ≈ 1.5 kW</center>

Step 5: Compare With Required Load

---


From earlier design calculation:

For 40 containers:

Required ≈ 4.2 kW

Your unit provides ≈ 1.5 kW

Therefore:

1.5 ≪ 4.2

The existing unit is insufficient even for 40 containers.

For 80 containers (≈ 8.5 kW required), it is completely inadequate.


Step 6:Required Developments
---

(A) Upgrade Refrigeration Capacity

Options:

Replace compressor with higher capacity unit.

Install an additional parallel AC unit.

Use industrial refrigeration system instead of comfort AC.


(B) Increase Evaporator Surface Area

Because:

Higher load → higher heat absorption required

Required:

• Larger evaporator coil

• Increased refrigerant flow rate

• Proper superheat control


(C) Improve Air Circulation

With double load:

• Air distribution becomes critical
• Dead zones may occur

Solutions:

• Install additional circulation fans
• Improve ducting
• Uniform container spacing


(D) Improve Insulation of Cool Room

Higher load increases:

• Transmission losses
• Compressor running time

Recommended:

• Increase insulation thickness
• Seal air leakage
• Use insulated doors

(E) Consider Pre-Cooling Stage

Instead of cooling from 40°C directly:

• Use plate heat exchanger at collection center
• Pre-cool milk to 25–30°C

This reduces refrigeration load significantly.


Using refrigerant-side energy balance based on experimental observations, the cooling capacity of the laboratory AC unit was calculated to be approximately 1.5 kW. This is significantly lower than the required cooling capacity of approximately 4.2 kW for 40 containers and 8.5 kW for 80 containers. Therefore, the existing unit is not suitable for the intended milk cooling application.
